### ISSUE TITLE
Random Opera Corruption

### ISSUE THREAD

**Issue by Degamisu** [ScratchAddons/ScratchAddons#7654]
### Description

Randomly (in a span of ~2 months) Scratch Addons corrupts on Opera. The repair button does not do anything, meaning it is either false flagged or broken beyond repair. It's very hard to tell what happened, and I'm not sure if I can find the logs.

### Steps to reproduce

No steps (random)

Could possibly be do with a memory error, where the computer's resources is taken up by other programs. Scratch Addons might not be able to handle this

### Environment

Browser: `Opera GX`
Scratch Addons version: [v1.38.4](https://scratchaddons.com/changelog/?utm_source=extension&utm_medium=popup&utm_campaign=v1.38.4#v1.38.4) 
Operating system: `Windows 11 23H2`

**Comment by CubesterYT** [comment]
Can you roughly estimate how many times it has occured within that time span?

**Comment by Degamisu** [comment]
> Can you roughly estimate how many times it has occured within that time span?

Thanks for the quick reply! It has only happened twice in the time span

**Comment by WorldLanguages** [comment]
> Scratch Addons might not be able to handle this

Indeed, I don't think an extension can get corrupted by itself. Maybe it can happen if we exceed some `chrome.storage` quota, but I'm not sure.

**Comment by rickbot-dot** [comment]
Just do yourself a favour and don't use Opera GX. It could be invading your privacy besides broken compatibility with extension (they're trying to get you to use the pointless feature known as Mods.)

**Comment by Degamisu** [comment]
> Just do yourself a favour and don't use Opera GX. It could be invading your privacy besides broken compatibility with a lot of extensions (they're trying to get you to use the pointless feature known as Mods.)

Yes I'm working on switching to just using firefox (I mostly use linux anyway).
